src/services/database/cascadeDelete.js

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable no-param-reassign */
+import schemas from '../../schemas';
+import relationshipFinder from '../utils/relationshipFinder';
+import patch from '../utils/patchRelationship';
+
+const promise = [];
+
+const traverseDelete = (record, table, schema) => {
```

Comment:
What happened to using `previousRelationship` as the third argument. Is that not valid for some reason?